transferwise/pipelinewise-transform-field
# Cosmetic Refactoring

## Context

For some expected upcoming features, I have done some cosmetic refactoring. 

### Changes
* Replace CircleCI with GHA, with check running for python 3.6, 3.7 and 3.8
* Use pytest instead of nose
* Add linting check and making pylint happy
*  Update README

## Checklist
- [x] I have considered the impact of this change and added a [Change Classification](
https://transferwise.atlassian.net/wiki/spaces/EKB/pages/1401189673/Change+Classifications+and+Expectations) Label (`change:standard`, `change:impactful`, `change:emergency`)
- [x] Change meets or does not compromise the [Baseline Security Requirements](https://transferwise.atlassian.net/wiki/spaces/EKB/pages/434929973/Baseline+Security+Requirements)
